README.md

```diff
@@ -31,6 +31,38 @@ Once enabled the bundle will expose several services, such as:
 
 ## Event Store
 
+By default the [InMemoryEventStore](https://github.com/broadway/broadway/blob/master/src/Broadway/EventStore/InMemoryEventStore.php) is
+used.
+
+Broadway provides a persisting event store implementation using `doctrine/dbal`
+in [broadway/event-store-dbal](https://github.com/broadway/event-store-dbal).
+
+This can be installed using composer:
+
+```
+$ composer require broadway/event-store-dbal
+```
+
+You need to configure this event store in you application:
+
+```xml
+<service id="my_dbal_event_store" class="Broadway\EventStore\DBALEventStore">
+    <argument type="service" id="doctrine.dbal.default_connection" />
+    <argument type="service" id="broadway.serializer.payload" />
+    <argument type="service" id="broadway.serializer.metadata" />
+    <argument>events</argument>
+    <argument>false</argument>
+    <argument type="service" id="broadway.uuid.converter" />
+</service>
+```
+
+And tell the Broadway bundle to use it:
+
+```yaml
+broadway:
```

Comment:
This goes for all configuration code-blocks.

For the service definitions I think it's already obvious enough, but it doesn't hurt adding `<!-- services.xml -->` there as well.